example-get-started/code/.github/workflows/cml.yaml

```diff
@@ -0,0 +1,27 @@
+name: CML Report
+on: push
+jobs:
+  run:
+    runs-on: [ubuntu-latest]
+    container: docker://dvcorg/cml:0-dvc2-base1
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
technically probably would also work:
```suggestion
          fetch-depth: 2
```

Reply:
Your changes are lost after the build. It should be included in the zip file but that zip file should be downloaded from AWS, and that also should be deployed by `deploy.sh` and I don't know why do we do this. (I fixed it in newer get-started.) We let the code take a world tour before making it into the repository. 

![Screen Shot 2021-05-12 at 17 57 12](https://user-images.githubusercontent.com/476310/117997342-8b8b1880-b34b-11eb-805f-2807e796607a.png)

![Screen Shot 2021-05-12 at 17 59 51](https://user-images.githubusercontent.com/476310/117997850-06ecca00-b34c-11eb-8b32-5701b8f47371.png)

I can fix this by simply copying the `src` dir? Would you or would I?

Reply:
I wonder - surely there was some reason why we were using `code.zip` from a deployment rather than `code` from this repo?

Reply:
I don't know. It should be a relic from some bygone era where code was written on clay tablets and transported on the back of dragons.

Reply:
From the history, it looks like it's been a deliberate consistent choice so I wouldn't remove it without feedback.

Reply:
no, it's not a relic :)

it was indeed a conscious decision -  if you read Get Started it's written in a way that you do `git init` yourself, add code, add data, etc. We needed some package and quick way to extract it to bootstrap the `src` dir in the repo.

Reply:
so probably need a separate ticket to add to `deploy.sh`? Or is that already handled by some other mechanism?

Reply:
oh wait it *is* in `deploy.sh` already.

Reply:
I think `code.zip` should be produced from the repo, not the other way around. It's possible to zip and upload to AWS at the end of `generate.sh`.

Reply:
it's already in the repo ... there is a publish.sh as far as I remeber

Reply:
Or produce `code.zip` in `build/` to upload.

Reply:
probably fixed by be88a706 :)

Reply:
correction - https://github.com/iterative/example-repos-dev/blob/master/example-get-started/deploy.sh - that's the script that does this :)

Reply:
getting this now. 

![Screen Shot 2021-05-12 at 18 44 34](https://user-images.githubusercontent.com/476310/118004419-1e2eb600-b352-11eb-8e50-59c0ed313289.png)

Reply:
Ah, sorry, had to run `deploy.sh` first.

Reply:
It looks OK now. 

![Screen Shot 2021-05-12 at 18 50 14](https://user-images.githubusercontent.com/476310/118005417-09065700-b353-11eb-949b-952104e0757e.png)

Reply:
I can push the result now